.github/workflows/cypress-repeat.yml

```
@@ -35,7 +35,10 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version-file: ".nvmrc"
-
+          
+      - name: Package
+        run: npm i
```

Should be installing from the lock file:
```suggestion
      - name: Package
        run: npm ci
```